FogFog-iOS/FogFog-iOS/Presentation/Login/ViewModel/LoginViewModel.swift

```diff
@@ -44,7 +44,7 @@ final class LoginViewModel: ViewModelType {
         input.kakaoButtonDidTap
             .flatMap(loginWithKakao)
             .subscribe { _ in
-                if UserDefaults.nickname == nil {
+                if UserInfo.nickname == "" {
```

Comment:
p3;
```suggestion
                if UserInfo.nickname.isEmpty {
```
요렇게 작성할 수도 있을 것 같아요